### PR TITLE
[public API] Abstractions documentation and API cleanup

### PR DESCRIPTION
**Pull request by vlada-shubina**
### Problem
fixes https://github.com/dotnet/templating/issues/3001 - Document all public members
fixes https://github.com/dotnet/templating/issues/3002 - Add CancellationToken for async public API methods

### Solution
- removed remaining unused public classes and members; moved IVersionSpecification from Abstractions to Utils
- added missing CancellationTokens to public async methods
- documentation in Abstractions
- nullable enable in public APIs of Orchestrator.RunnableProjects

### Checks:
- [ ] Added unit tests - NA
- [ ] Added `#nullable enable` to all the modified files [?](https://github.com/dotnet/templating/wiki/Contributing#coding-style) - NA

### CODE REVIEW

**Review by DavidKarlas — CHANGES_REQUESTED** [review]
Some classes in Abstractions don't have #nullable enable, intentional?

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 1]
```diff
@@ -2,23 +2,82 @@
...
 
+        /// <summary>
+        /// Dry runs <paramref name="template"/> with given parameters.
+        /// </summary>
+        /// <param name="environmentSettings">template engine environment settings.</param>
+        /// <param name="template">template to dry run.</param>
+        /// <param name="parameters">template parameters.</param>
+        /// <param name="componentManager">component manager to use.</param>
+        /// <param name="targetDirectory">target output directory to generate template to.</param>
+        /// <returns><see cref="ICreationEffects"/> containing file changes, post actions and primary outputs that would have been done after template generation.</returns>
```
[the remark this answers is not in the document]
Comment (vlada-shubina) [reply src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:43]: `Contents` is always empty now, I add Obsolete message for it. I won't change description here.

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 2]
```diff
@@ -2,23 +2,82 @@
...
+        /// <param name="config">configuration file.</param>
+        /// <param name="template">loaded template.</param>
+        /// <param name="localeConfig">localization configuration file.</param>
+        /// <param name="hostTemplateConfigFile">host template configuration file.</param>
+        /// <param name="baselineName">baseline to load.</param>
+        /// <returns>true if <paramref name="template"/> can be read, false otherwise.</returns>
         bool TryGetTemplateFromConfigInfo(IFileSystemInfo config, out ITemplate template, IFileSystemInfo localeConfig, IFile hostTemplateConfigFile, string baselineName = null);
 
+        /// <summary>
+        /// Gets <see cref="ITemplate"/>s from given <see cref="IFileSystemInfo" /> from given mount point <paramref name="source"/>.
```
[the remark this answers is not in the document]
Comment (vlada-shubina) [reply src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:66]: `Scans mount point <paramref name="source"/> for templates and returns list of <see cref="ITemplate"/>s found.`

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 3]
```diff
@@ -2,23 +2,82 @@
...
+        /// <param name="environmentSettings">template engine environment settings.</param>
+        /// <param name="template">template to dry run.</param>
+        /// <param name="parameters">template parameters.</param>
+        /// <param name="componentManager">component manager to use.</param>
+        /// <param name="targetDirectory">target output directory to generate template to.</param>
+        /// <returns><see cref="ICreationEffects"/> containing file changes, post actions and primary outputs that would have been done after template generation.</returns>
         ICreationEffects GetCreationEffects(IEngineEnvironmentSettings environmentSettings, ITemplate template, IParameterSet parameters, IComponentManager componentManager, string targetDirectory);
 
+        /// <summary>
+        /// Returns <see cref="IParameterSet"/> for <paramref name="template"/>.
```
[the remark this answers is not in the document]
Comment (vlada-shubina) [reply src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:47]:
```
        /// Returns <see cref="IParameterSet"/> for <paramref name="template"/>.
        /// This method returns the list of input parameters that can be set by the host / Edge before running the template.
        /// After setting the values the host should pass <see cref="IParameterSet"/> to <see cref="GetCreationEffectsAsync(IEngineEnvironmentSettings, ITemplate, IParameterSet, string, CancellationToken)"/> or <see cref="CreateAsync(IEngineEnvironmentSettings, ITemplate, IParameterSet, string, CancellationToken)"/> methods.
        /// Host may use <see cref="ConvertParameterValueToType(IEngineEnvironmentSettings, ITemplateParameter, string, out bool)"/> to convert input value to required parameter type.
```

File: src/Microsoft.TemplateEngine.Abstractions/IBaselineInfo.cs [hunk 4]
```diff
@@ -1,14 +1,25 @@
...
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
+
 using System.Collections.Generic;
 
 namespace Microsoft.TemplateEngine.Abstractions
 {
+    /// <summary>
+    /// Defines baseline information.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IBaselineInfo.cs:11]: I don't know what baseline information means. Should we be a bit more descriptive here?

File: src/Microsoft.TemplateEngine.Abstractions/IFileChange.cs [hunk 5]
```diff
@@ -24,6 +26,7 @@ public interface IFileChange
         /// <summary>
         /// Gets the file content.
         /// </summary>
+        [Obsolete("file contents are not returned in this version, the property always empty byte array.")]
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IFileChange.cs:29]: A bit of cleaning up:
```suggestion
        [Obsolete("File contents are no longer available. This property always returns an empty byte array.")]
```

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 6]
```diff
@@ -2,23 +2,88 @@
...
+        /// <param name="config">configuration file.</param>
+        /// <param name="template">loaded template.</param>
+        /// <param name="localeConfig">localization configuration file.</param>
+        /// <param name="hostTemplateConfigFile">host template configuration file.</param>
+        /// <param name="baselineName">baseline to load.</param>
+        /// <returns>true if <paramref name="template"/> can be read, false otherwise.</returns>
         bool TryGetTemplateFromConfigInfo(IFileSystemInfo config, out ITemplate template, IFileSystemInfo localeConfig, IFile hostTemplateConfigFile, string baselineName = null);
 
+        /// <summary>
+        /// Scans mount point <paramref name="source"/> for templates and returns list of <see cref="ITemplate"/>s found.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:72]:
```suggestion
        /// Scans the given mount point <paramref name="source"/> for templates and returns the list of <see cref="ITemplate"/>s that are found found.
```

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 7]
```diff
@@ -2,23 +2,88 @@
...
 {
+    /// <summary>
+    /// Defines template generator supported by template engine.
+    /// This interface should be implemented to create generator loadable by template engine infrastructure.
+    /// </summary>
     public interface IGenerator : IIdentifiedComponent
     {
-        Task<ICreationResult> CreateAsync(IEngineEnvironmentSettings environmentSettings, ITemplate template, IParameterSet parameters, IComponentManager componentManager, string targetDirectory);
+        /// <summary>
+        /// Generates <paramref name="template"/> with given parameters.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:18]:
```suggestion
        /// Generates the <paramref name="template"/> with given parameters.
```

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 8]
```diff
@@ -2,23 +2,88 @@
...
+        Task<ICreationResult> CreateAsync(
+            IEngineEnvironmentSettings environmentSettings,
+            ITemplate template,
+            IParameterSet parameters,
+            string targetDirectory,
+            CancellationToken cancellationToken);
 
-        ICreationEffects GetCreationEffects(IEngineEnvironmentSettings environmentSettings, ITemplate template, IParameterSet parameters, IComponentManager componentManager, string targetDirectory);
+        /// <summary>
+        /// Dry runs <paramref name="template"/> with given parameters.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:34]:
```suggestion
        /// Dry runs the <paramref name="template"/> with given parameters.
```

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 9]
```diff
@@ -2,23 +2,88 @@
...
+        /// <returns><see cref="ICreationEffects"/> containing file changes, post actions and primary outputs that would have been done after template generation.</returns>
+        Task<ICreationEffects> GetCreationEffectsAsync(
+            IEngineEnvironmentSettings environmentSettings,
+            ITemplate template,
+            IParameterSet parameters,
+            string targetDirectory,
+            CancellationToken cancellationToken);
 
+        /// <summary>
+        /// Returns <see cref="IParameterSet"/> for <paramref name="template"/>.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:50]:
```suggestion
        /// Returns a <see cref="IParameterSet"/> for the given <paramref name="template"/>.
```

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 10]
```diff
@@ -2,23 +2,88 @@
...
+        /// After setting the values the host should pass <see cref="IParameterSet"/> to <see cref="GetCreationEffectsAsync(IEngineEnvironmentSettings, ITemplate, IParameterSet, string, CancellationToken)"/> or <see cref="CreateAsync(IEngineEnvironmentSettings, ITemplate, IParameterSet, string, CancellationToken)"/> methods.
+        /// Host may use <see cref="ConvertParameterValueToType(IEngineEnvironmentSettings, ITemplateParameter, string, out bool)"/> to convert input value to required parameter type.
+        /// </summary>
+        /// <param name="environmentSettings">template engine environment settings.</param>
+        /// <param name="template">template to get parameters from.</param>
+        /// <returns><see cref="IParameterSet"/> with parameters available in <paramref name="template"/>.</returns>
         IParameterSet GetParametersForTemplate(IEngineEnvironmentSettings environmentSettings, ITemplate template);
 
+        /// <summary>
+        /// Gets <see cref="ITemplate"/> from  given <see cref="IFileSystemInfo" /> configuration entry.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:61]:
```suggestion
        /// Gets an <see cref="ITemplate"/> from the given <see cref="IFileSystemInfo" /> configuration entry.
```

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 11]
```diff
@@ -2,23 +2,88 @@
...
+        /// <summary>
+        /// Scans mount point <paramref name="source"/> for templates and returns list of <see cref="ITemplate"/>s found.
+        /// </summary>
+        /// <param name="source">the mount point to load templates from.</param>
+        /// <param name="localizations">localization information for templates.</param>
+        /// <returns>the list of templates found in <paramref name="source"/> mount point.</returns>
         IList<ITemplate> GetTemplatesAndLangpacksFromDir(IMountPoint source, out IList<ILocalizationLocator> localizations);
 
+        /// <summary>
+        /// Converts <paramref name="untypedValue"/> to type defined for <paramref name="parameter"/>.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:80]:
```suggestion
        /// Converts the given <paramref name="untypedValue"/> to the type of <paramref name="parameter"/>.
```

File: src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs [hunk 12]
```diff
@@ -2,23 +2,88 @@
...
+        /// </summary>
+        /// <param name="environmentSettings">template engine environment settings.</param>
+        /// <param name="parameter">template parameter defining the type.</param>
+        /// <param name="untypedValue">the value to be converted to parameter type.</param>
+        /// <param name="valueResolutionError">true if value could not be converted, false otherwise.</param>
+        /// <returns>the converted value to type of <paramref name="parameter"/> or null if value cannot be converted.</returns>
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IGenerator.cs:86]:
```suggestion
        /// <returns> <paramref name="untypedValue"/> after it was converted to the type of <paramref name="parameter"/>, or null if the value cannot be converted.</returns>
```

File: src/Microsoft.TemplateEngine.Abstractions/IIdentifiedComponent.cs [hunk 13]
```diff
@@ -5,8 +5,14 @@
 
 namespace Microsoft.TemplateEngine.Abstractions
 {
+    /// <summary>
+    /// Defines component loadable by <see cref="IComponentManager"/>.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IIdentifiedComponent.cs:9]:
```suggestion
    /// Defines a component loadable by <see cref="IComponentManager"/>.
```

File: src/Microsoft.TemplateEngine.Abstractions/IIdentifiedComponent.cs [hunk 14]
```diff
@@ -5,8 +5,14 @@
...
+    /// </summary>
     public interface IIdentifiedComponent
     {
+        /// <summary>
+        /// Gets identifier of the component. Should be unique.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IIdentifiedComponent.cs:14]:
```suggestion
        /// Gets the identifier of the component. Should be unique.
```

File: src/Microsoft.TemplateEngine.Abstractions/IParameterSet.cs [hunk 15]
```diff
@@ -5,14 +5,27 @@
 
 namespace Microsoft.TemplateEngine.Abstractions
 {
+    /// <summary>
+    /// Defines template parameter set.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IParameterSet.cs:9]:
```suggestion
    /// Defines a set of template parameters.
```

File: src/Microsoft.TemplateEngine.Abstractions/IParameterSet.cs [hunk 16]
```diff
@@ -5,14 +5,27 @@
...
+    /// </summary>
     public interface IParameterSet
     {
+        /// <summary>
+        /// Gets enumerator to template parameter definitions.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IParameterSet.cs:14]:
```suggestion
        /// Gets an enumerator iterating through the parameter definitions of the template.
```

File: src/Microsoft.TemplateEngine.Abstractions/IParameterSet.cs [hunk 17]
```diff
@@ -5,14 +5,27 @@
...
+        /// </summary>
         IEnumerable<ITemplateParameter> ParameterDefinitions { get; }
 
-        IEnumerable<string> RequiredBrokerCapabilities { get; }
-
+        /// <summary>
+        /// Gets collection of template parameters and their values.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IParameterSet.cs:19]:
```suggestion
        /// Gets a collection of template parameters and their values.
```

File: src/Microsoft.TemplateEngine.Abstractions/IParameterSet.cs [hunk 18]
```diff
@@ -5,14 +5,27 @@
...
+        /// </summary>
         IDictionary<ITemplateParameter, object> ResolvedValues { get; }
 
+        /// <summary>
+        /// Gets parameter definition with <paramref name="name"/>.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IParameterSet.cs:24]:
```suggestion
        /// Gets the parameter definition with the specified <paramref name="name"/>.
```

File: src/Microsoft.TemplateEngine.Abstractions/IParameterSymbolLocalizationModel.cs [hunk 19]
```diff
@@ -7,8 +7,14 @@
 
 namespace Microsoft.TemplateEngine.Abstractions
 {
+    /// <summary>
+    /// Defines parameter localization model.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IParameterSymbolLocalizationModel.cs:11]:
```suggestion
    /// Defines a parameter localization model.
```

File: src/Microsoft.TemplateEngine.Abstractions/IPostAction.cs [hunk 20]
```diff
@@ -8,6 +8,9 @@
 
 namespace Microsoft.TemplateEngine.Abstractions
 {
+    /// <summary>
+    /// Defines post action supported by the template.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/IPostAction.cs:12]:
```suggestion
    /// Defines a post action supported by the template.
```

File: src/Microsoft.TemplateEngine.Abstractions/ITemplate.cs [hunk 21]
```diff
@@ -5,16 +5,34 @@
...
+        /// </summary>
         IFileSystemInfo LocaleConfiguration { get; }
 
+        /// <summary>
+        /// Gets directory with template source files.
+        /// </summary>
         IDirectory TemplateSourceRoot { get; }
 
+        /// <summary>
+        /// Indicates whether template should be created in subfolder in output directory.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/ITemplate.cs:34]:
```suggestion
        /// Indicates whether the template should be created in a subfolder under the output directory.
```

File: src/Microsoft.TemplateEngine.Abstractions/Mount/FileSystemInfoKind.cs [hunk 22]
```diff
@@ -3,9 +3,19 @@
 
 namespace Microsoft.TemplateEngine.Abstractions.Mount
 {
+    /// <summary>
+    /// Defines kind of <see cref="IFileSystemInfo"/> entry.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/Mount/FileSystemInfoKind.cs:7]:
```suggestion
    /// Defines the kind of a <see cref="IFileSystemInfo"/> entry.
```

File: src/Microsoft.TemplateEngine.Abstractions/Mount/FileSystemInfoKind.cs [hunk 23]
```diff
@@ -3,9 +3,19 @@
...
+    /// </summary>
     public enum FileSystemInfoKind
     {
+        /// <summary>
+        /// Entry is file.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/Mount/FileSystemInfoKind.cs:12]:
```suggestion
        /// Entry is a file.
```

File: src/Microsoft.TemplateEngine.Abstractions/Mount/FileSystemInfoKind.cs [hunk 24]
```diff
@@ -3,9 +3,19 @@
...
+        /// </summary>
         File,
+
+        /// <summary>
+        /// Entry is directory.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/Mount/FileSystemInfoKind.cs:17]:
```suggestion
        /// Entry is a directory.
```

File: src/Microsoft.TemplateEngine.Abstractions/Mount/IDirectory.cs [hunk 25]
```diff
@@ -6,12 +6,33 @@
...
 namespace Microsoft.TemplateEngine.Abstractions.Mount
 {
+    /// <summary>
+    /// Defines directory <see cref="IFileSystemInfo"/> entry.
+    /// </summary>
     public interface IDirectory : IFileSystemInfo
     {
-        IEnumerable<IFileSystemInfo> EnumerateFileSystemInfos(string patten, SearchOption searchOption);
+        /// <summary>
+        /// Enumerates <see cref="IFileSystemInfo"/> entries in the directory.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/Mount/IDirectory.cs:15]:
```suggestion
        /// Enumerates the <see cref="IFileSystemInfo"/> entries in the directory.
```

File: src/Microsoft.TemplateEngine.Abstractions/Mount/IDirectory.cs [hunk 26]
```diff
@@ -6,12 +6,33 @@
...
+        /// </summary>
+        /// <param name="pattern">Matching pattern for the entries, see <see cref="Directory.EnumerateFileSystemEntries(string, string, SearchOption)"/> for more details.</param>
+        /// <param name="searchOption">Matching pattern for the entries, see <see cref="Directory.EnumerateFileSystemEntries(string, string, SearchOption)"/> for more details.</param>
+        /// <returns>The enumerator to <see cref="IFileSystemInfo"/> entries in the directory.</returns>
+        IEnumerable<IFileSystemInfo> EnumerateFileSystemInfos(string pattern, SearchOption searchOption);
 
+        /// <summary>
+        /// Enumerates <see cref="IFile"/> entries in the directory.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/Mount/IDirectory.cs:23]:
```suggestion
        /// Enumerates the <see cref="IFile"/> entries in the directory.
```

File: src/Microsoft.TemplateEngine.Abstractions/Mount/IDirectory.cs [hunk 27]
```diff
@@ -6,12 +6,33 @@
...
+        /// </summary>
+        /// <param name="pattern">Matching pattern for the entries, see <see cref="Directory.EnumerateFiles(string, string, SearchOption)"/> for more details.</param>
+        /// <param name="searchOption">Matching pattern for the entries, see <see cref="Directory.EnumerateFiles(string, string, SearchOption)"/> for more details.</param>
+        /// <returns>The enumerator to <see cref="IFile"/> entries in the directory.</returns>
         IEnumerable<IFile> EnumerateFiles(string pattern, SearchOption searchOption);
 
+        /// <summary>
+        /// Enumerates <see cref="IDirectory"/> entries in the directory.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/Mount/IDirectory.cs:31]:
```suggestion
        /// Enumerates the <see cref="IDirectory"/> entries in the directory.
```

File: src/Microsoft.TemplateEngine.Abstractions/Mount/IFile.cs [hunk 28]
```diff
@@ -5,8 +5,15 @@
 
 namespace Microsoft.TemplateEngine.Abstractions.Mount
 {
+    /// <summary>
+    /// Defines file <see cref="IFileSystemInfo"/> entry.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/Mount/IFile.cs:9]:
```suggestion
    /// Defines a file <see cref="IFileSystemInfo"/> entry.
```

File: src/Microsoft.TemplateEngine.Abstractions/Mount/IFileSystemInfo.cs [hunk 29]
```diff
@@ -3,18 +3,41 @@
 
 namespace Microsoft.TemplateEngine.Abstractions.Mount
 {
+    /// <summary>
+    /// Defines file system entry.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/Mount/IFileSystemInfo.cs:7]:
```suggestion
    /// Defines a file system entry.
```

File: src/Microsoft.TemplateEngine.Abstractions/TemplateFiltering/MatchInfo.cs [hunk 30]
```diff
@@ -12,6 +12,12 @@ namespace Microsoft.TemplateEngine.Abstractions.TemplateFiltering
     /// </summary>
     public class MatchInfo
     {
+        /// <summary>
+        /// Creates <see cref="MatchInfo"/> instance.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Abstractions/TemplateFiltering/MatchInfo.cs:16]:
```suggestion
        /// Creates a <see cref="MatchInfo"/> instance.
```
or
```suggestion
        /// Creates an instance of <see cref="MatchInfo"/>.
```

File: src/Microsoft.TemplateEngine.Utils/IVersionSpecification.cs [hunk 31]
```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+namespace Microsoft.TemplateEngine.Utils
+{
+    /// <summary>
+    /// Defines version specification used in template definition.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Utils/IVersionSpecification.cs:9]:
```suggestion
    /// Defines a version specification used in a template definition.
```

File: src/Microsoft.TemplateEngine.Utils/IVersionSpecification.cs [hunk 32]
```diff
@@ -0,0 +1,20 @@
...
+    /// </summary>
+    public interface IVersionSpecification
+    {
+        /// <summary>
+        /// Checks is version is valid.
```
Comment (bekir-ozturk) [inline src/Microsoft.TemplateEngine.Utils/IVersionSpecification.cs:14]:
```suggestion
        /// Checks whether the specified version is valid.
```

**Review by bekir-ozturk — APPROVED** [review]

File: src/Microsoft.TemplateEngine.Abstractions/IBaselineInfo.cs (hunk 4 above)
Comment (vlada-shubina) [reply src/Microsoft.TemplateEngine.Abstractions/IBaselineInfo.cs:11]: I added short explanation  `Baseline configuration is a set of predefined template parameters and their values.`, however this term is quite well known.

**Pull request merged**